test/unit/transform/ferc1_test.py

```diff
@@ -128,21 +134,32 @@ def test_wide_to_tidy_xbrl():
     pd.testing.assert_frame_equal(df_out, df_expected)
 
 
-def test_wide_to_tidy_xbrl_fail():
+def test_wide_to_tidy_fail():
     """Test the :func:`wide_to_tidy_xbrl` fails with a bad rename."""
-    params = WideToTidyXbrl(**{"idx_cols": ["idx"], "value_types": ["test_value"]})
+    params = WideToTidy(
+        **{
+            "idx_cols": ["idx"],
+            "value_types": ["test_value"],
+            "stacked_column_name": "xbrl_factoid",
+        }
```

Comment:
Given that we're already enumerating all of the dictionary elements (both here and in `test_wide_to_tidy_rename()` below and `test_wide_to_tidy()` above) why not just pass them as normal arguments instead of constructing and then immediately unpacking the dictionary?  I.e.

```py
params = WideToTidy(
    idx_cols=["idx"],
    value_types=["test_value"],
    stacked_column_name="xbrl_factoid",
)
```

Reply:
we certainly could.. but does it really matter?